### ISSUE TITLE
Support for elevenlabs.io

### ISSUE THREAD

**Issue by pirj** [omnivore-app/obsidian-omnivore#268]
Omnivore was acquired by Elevenlabs, and they provide an upgrade path.
It is possible to create an API key on elevenlabs.io:
![image](https://github.com/user-attachments/assets/d452ad66-3e24-4b49-aa52-5a4b19489153)
Will this plugin support syncing from elevenlabs.io?

**Comment by digitalkeegan** [comment]
Yeah came here to say the same thing. I love using Omnivore in Obsidian. It's the only reason I even use Omnivore. I'd love for this plugin to transition!

**Comment by pirj** [comment]
An alternative approach is to (on iOS) to use the [ToMarkdown](https://github.com/macsplit/tomarkdown) Safari extension that turns a page into Markdown, which you can Share to Obsidian app directly.
That way, though, no highlights, no template etc, and you have to type in the file name.
But also you don’t have to wait for the Obsidian extension to sync when you start it (I know there’s an option to turn this off).

I apologise for the offtopic.
